### ISSUE TITLE
GradientProgressBar couldn't be subclassed

### ISSUE THREAD

**Issue by iTofu** [fxm90/GradientProgressBar#5]
`GradientProgressBar` couldn't be subclassed outside the its module for now, cause it's a `public` class. I think it would be better to change `public` class to `open` one.
```swift
// error message
// Cannot inherit from non-open class 'GradientProgressBar' outside of its defining module


public class GradientProgressBar: UIProgressView {}
// ->
open class GradientProgressBar: UIProgressView {}
```

**Comment by fxm90** [comment]
Hey, I've created a branch `feature/005-allow-subclassing` and changed the class access control to `open`. Could you please verify that this branch solves your problem? :) If so, I'm gonna create a new release afterwards. Thx

**Comment by iTofu** [comment]
I checked it, it works will, and thank you 👍

**Comment by fxm90** [comment]
Cool, I‘ll release this as a new version soonish and afterwards close this issue :)

**Comment by fxm90** [comment]
fixed in version 1.2.7
